feat: add mohamed-bishr domain configuration

MohamedWaelBishr opened the pull request:
<!--
!!!
YOU MUST FILL OUT THIS TEMPLATE FOR YOUR PR TO BE ACCEPTED.
IF YOU DO NOT FILL OUT THIS PR TEMPLATE TO ITS ENTIRETY, YOUR PR WILL BE IMMEDIATELY DENIED.
!!!
-->

# Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain **MUST** pass **ALL** the indicated requirements below.

<!-- Change each checkbox to [x] to mark it as checked. Do not keep the spaces between the parentheses. -->
- [x] I have **read** and **understood** the [Terms of Service](https://is-a.dev/terms). <!-- Your domain MUST follow the TOS to be approved. -->
- [x] My domain is **not** for commercial use. <!-- e.g. Your domain should not be selling products & services -->
- [x] My file is in the `domains` directory and has the `.json` file extension.
- [x] My file's name is lowercased and alphanumeric. <!-- Your file's name is yourname.json, not YourName.json or your_name.json. -->
- [x] My website is **reachable**.
- [x] I have **completed** my website. <!-- We do not permit simple "Hello, world!" websites. -->
- [x] My website is related to **software development**. <!-- We don't permit websites for domains unrelated to development (e.g. Discord servers, gaming guilds, etc.) -->
- [x] I have provided sufficient contact information in the `owner` key. <!-- Provide your email in the `email` field or another platform (e.g., X, Discord) for contact. -->
- [x] I have followed and read the [documentation](https://docs.is-a.dev).
- [x] I understand my domain will be removed if I violate the [Terms of Service](https://is-a.dev/terms).

# Website Preview
<!-- Provide a link or preview of your website below. If you can't make the website visible, then a screenshot of the website also works. -->

MohamedWaelBishr committed:
feat: add mohamed-bishr domain configuration

MohamedWaelBishr committed:
feat: migrate mohamed-bishr domain configuration to new file format

MohamedWaelBishr committed:
Merge branch 'is-a-dev:main' into main

21Z commented on domains/mohamedbishr.json:
```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "mohamed-bishr",
```
provided username does not match github username

MohamedWaelBishr replied:
Done

MohamedWaelBishr committed:
Updates username in mohamedbishr.json

Changes username from mohamed-bishr to MohamedWaelBishr to reflect correct account name

MohamedWaelBishr committed:
Merge branch 'main' into main
